blivet/util.py

```diff
@@ -450,6 +456,8 @@ def get_cow_sysfs_path(dev_path, dev_sysfsPath):
 
 def match_path_context(path):
```

Comment:
All calls to these function are "protected" by checking the SELinux flag first, so we should never raise the `RuntimeError` below, but these are still "public" functions so it is probably good idea to add the check just to be safe.